vertical-pod-autoscaler/pkg/recommender/logic/recommender_test.go

```diff
@@ -115,3 +117,53 @@ func TestControlledResourcesFilteredDefault(t *testing.T) {
 	assert.Contains(t, recommendedResources[containerName].LowerBound, model.ResourceCPU)
 	assert.Contains(t, recommendedResources[containerName].UpperBound, model.ResourceCPU)
 }
+
+func TestMapToListOfRecommendedContainerResources(t *testing.T) {
+	cases := []struct {
+		name         string
+		resources    RecommendedPodResources
+		expectedLast []string
+	}{
+		{
+			name: "All recommendations sorted",
+			resources: RecommendedPodResources{
+				"a-container": RecommendedContainerResources{Target: model.Resources{model.ResourceCPU: model.CPUAmountFromCores(1), model.ResourceMemory: model.MemoryAmountFromBytes(1000)}},
```

Comment:
Please use different recommendations for different containers so that the test could tell if we were changing which recommendation is for which container